sample_data/defaultOrganization.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "_id": "6637904485008f171cf29924",
+    "status": "ACTIVE",
+    "members": ["66378abd85008f171cf2990d"],
+    "admins": ["66378abd85008f171cf2990d"],
+    "groupChats": [],
+    "posts": [],
+    "pinnedPosts": [],
+    "membershipRequests": [],
+    "blockedUsers": [],
+    "name": "Hope Foundation",
```

Comment:
Change to `Default Organization` so that it's more easily understood that it needs to be changed